tests/schemas/test_census_individual.json

```diff
@@ -0,0 +1,11778 @@
+{
```

Comment:
Again here, I don't think this should be included as part of the repo. Either we need to write something to pull these in or remove them entirely.